inc/commonitilobject.class.php

```diff
@@ -6711,7 +6711,6 @@ function viewEditSubitem" . $this->fields['id'] . "$rand(e, itemtype, items_id,
 
          $criteria = [
             'SELECT'   => 'actiontime',
```

Comment:
May be we should directly request the SUM.

Reply:
I agree. I also agree with the distinct removal

Reply:
We thought about it but we didn't want to alter to much code to get it accepted ASAP :) 
@xacobofg is making the SUM.